### DISCUSSION TITLE
📖Resources to Read on Binary trees🔖

### DISCUSSION THREAD

**Discussion by Hezethegamer** [the1Riddle/binary_trees#6] [Q&A] [answered]
can i get recourses to this?

**Comment by the1Riddle** [comment, marked as answer]
Year sure, 
here are the link to some resources you can read on binary trees,

**Youtube Videos**

link 1: [How to implement a tree in C](https://youtu.be/UbhlOk7vjVY?si=J2ZmuBZPzER1ZSng)
link 2: [Binary Trees, Part 1](https://youtu.be/76dhtgZt38A?si=ZyUOAdhctvVueobk)
link 3: [Data structures: Introduction to Trees](https://www.youtube.com/watch?v=qH6yxkw0u78)
link 4: [Data structures: Binary Tree](https://www.youtube.com/watch?v=H5JubkIy_p8&t=2s)
link 5: [Data structures: Binary Search Tree](https://youtu.be/pYT9F8_LFTM?si=9eq9GN44VUTgkZkd)
link 6: [Find height of a binary tree](https://www.youtube.com/watch?v=_pnqMz5nrRs)

**Reply by Hezethegamer** [reply]
thanks

**Reply by the1Riddle** [reply]
no problem ^_^

**Comment by the1Riddle** [comment]
some `Articles` you can also read

**Remember also to check the resources from Alx**

1. [Binary Tree Data Structure](https://www.geeksforgeeks.org/binary-tree-data-structure/)
2. [Binary Search Tree](https://www.geeksforgeeks.org/binary-search-tree-data-structure/)
3. [Time & Space Complexity of Binary Tree operation](https://iq.opengenus.org/time-complexity-of-binary-tree/)
4. [Balanced Binary Tree and How to Check it](https://www.digitalocean.com/community/tutorials/balanced-binary-tree-check)

**Comment by rezzcode** [comment]
- check out this

some more resources you would find useful during your journey in alx are [here](https://alx-feb-resources.notion.site/ALX-Resources-1660799bece74b2f9afb1bcc24dff313)
